shit.php

```diff
@@ -0,0 +1 @@
+<?php echo "shit"; ?>
```

Comment:
```suggestion
<?= "shit";
```
This can be simplified :)